### ISSUE TITLE
add OTOBO to the project list

### ISSUE THREAD

**Issue by bschmalhofer** [hakonhagland/perl-projects#1]
OTOBO is a Perl-based open source ticketing system.  The code is in Github: https://github.com/RotherOSS/otobo . The project home page is at https://otobo.org

**Comment by hakonhagland** [comment]
Excellent! Could you provide a project page like [this](https://github.com/hakonhagland/perl-qt5-project) that links to your project and summarizes what contributions you would like to have? Then I will include a link to that in my list of projects!

**Comment by bschmalhofer** [comment]
You could use https://otobo.de/en/community/ as a starting point.

**Comment by hakonhagland** [comment]
@bschmalhofer Ok that looks good to me! Thanks for the contribution, added the link to the list projects.

**Comment by hakonhagland** [comment]
After closer consideration I think it would be nice if you provided the project page instead of linking directly to the OTOBO page. I think my idea is this: 
- Provide a unified interface to this repository. That is: clicking a link in the list will always bring you to a project page, where a more specific information is given. 
- The contributor will know which person that requested the contribution, namely the maintainer of the project page.
- By linking to the project page it is explicitly said what should be the aim of this project. I believe this is important, because it shows that the maintainer is involved in the project and is willing to collaborate.. This will make the contributor more interested in contributing..
- This will also be the place where the contributor can ask question regarding the project..

**Comment by bschmalhofer** [comment]
I think that I kind of understand your approach, providing well defined tasks so that people can start contributing. For OTOBO however, the relevant contributors have a different profile. Mostly they are users that are already using OTOBO and have a specific itch to scratch. 
Of course anybody providing a pull request for one of the issues in Github is very welcome too.

**Comment by hakonhagland** [comment]
> providing well defined tasks so that people can start contributing

Yes that is correct! Ok, removing OTOBO from the list for now then..

**Comment by sabahmax-dev** [comment]
Thanks for the suggestion! I’ve opened [#2](https://github.com/hakonhagland/perl-projects/pull/2) to add OTOBO to the project list. It adds OTOBO, a Perl-based open source ticketing system, with links to its [GitHub repository](https://github.com/RotherOSS/otobo) and [homepage](https://otobo.org). Fixes #1.
